simple/rdm_shared_av.c

```diff
@@ -44,6 +44,7 @@
 static int init_fabric(void)
 {
 	int ret;
+	int other;
```

Comment:
nit: can we call this ret2 or err?  When I saw 'other' I thought of it as a boolean